integration-tests/data/golist.json

```diff
@@ -0,0 +1,1115 @@
+{
+  "version": "v1",
+  "main": "github.com/arajkumar/hello-vuln",
+  "packages": [
```

Comment:
Do we need such a large file for testing. This seems more like a perf test.

Reply:
will make it short